app/jobs/alert_fill_contribution_description_job.rb

```diff
@@ -3,10 +3,14 @@ class AlertFillContributionDescriptionJob < ApplicationJob
 
   def perform(user_id)
     user = User.find(user_id)
-    contributions = user.contributions.received 
+    contributions = user.contributions.received.without_description
 
     unless contributions.empty?
-      NotificationMailer.notify_fill_contribution_description(user, contributions.length).deliver_later
+      NotificationMailer.notify_fill_contribution_description(
+        user: user,
+        contributions_total: contributions.length,
+        contributions_links: contributions.pluck(:id, :link)
```

Comment:
here you have the relation `contributions`, and you call, length, and pluck, you will end up hitting the db twice

an approach to avoid this would be

```suggestion
        contributions_total: contributions.length,
        contributions_links: contributions.map { [_1.id, _1.link] }
```

or 

```suggestion
        contributions_link = contributions.pluck(:id, :link)
        contributions_total: contributions_link.lenth,
        contributions_links: contributions_link
```

Reply:
Fixed it! Ty